src/components/ComboButton/_index.scss

```diff
@@ -4,6 +4,9 @@
 /// @copyright IBM Security 2019
 ////
 
+@import 'carbon-components/scss/components/button/button';
```

Comment:
What are your thoughts on importing our `Button` and `Dropdown` styles here versus Carbon's?

Reply:
Sure I can do that